src/components/navbar/Dropdown.js

```diff
@@ -26,16 +26,26 @@ class Dropdown extends Component<Props, State> {
 
   handleClick = () => {
     this.setState(state => ({ isSelected: !state.isSelected }));
-  }
+  };
 
   handleClickOutside = () => {
     this.setState({ isSelected: false });
   };
 
   handleLogoutClick = (e) => {
+    require('google-client-api')().then((gapi) => {
```

Comment:
Any particular reason why to import the package here?
And use `await` instead of `then` to avoid nesting.
You can rewrite 
```
somePromise.then((data) => { ... });
```
to
```
const data = await somePromise;
```